bin/localicious-install

```diff
@@ -30,12 +30,7 @@ loadYaml(localiciousConfig).onSuccess(localiciousConfig => {
       .flatMap(configuration => {
         const { path, options } = configuration;
 
-        return validateAndRender(
-          path,
-          outputPath,
-          options.languages,
-          options.platforms
-        );
+        return validateAndRender(path, outputPath, options.languages, options.platforms);
```

Comment:
Quite a few of these formatting changes? Do we have different formatter settings, did you change something to the configuration, or did I change something without reformatting everything?